src/Control/SimpleResourceURLGenerator.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SilverStripe\Control;
+
+use SilverStripe\Core\Manifest\ResourceURLGenerator;
+
+/**
+ * Generate URLs assuming that BASE_PATH is also the webroot
+ * Standard SilverStripe 3 operation
+ */
+class SimpleResourceURLGenerator implements ResourceURLGenerator
+{
+    /*
+     * @var string
+     */
+    private $nonceStyle;
+
+    /*
+     * Get the style of nonce-suffixes to use, or null if disabled
+     *
+     * @return string|null
+     */
+    public function getNonceStyle()
+    {
+        return $this->nonceStyle;
+    }
+
+    /*
+     * Set the style of nonce-suffixes to use, or null to disable
+     * Currently only "mtime" is allowed
+     *
+     * @param string|null $nonceStyle The style of nonces to apply, or null to disable
+     */
+    public function setNonceStyle($nonceStyle)
+    {
+        if ($nonceStyle && $nonceStyle !== 'mtime') {
+            throw new InvalidArgumentException('The only allowed NonceStyle is mtime');
+        }
+        $this->nonceStyle = $nonceStyle;
+    }
+
+    /**
+     * Return the URL for a resource, prefixing with Director::baseURL() and suffixing with a nonce
+     *
+     * @inherit
```

Comment:
This isn't actually a phpdoc feature. There is `@inheritdoc` but that only inherits the description, not the args / return.